### DISCUSSION TITLE
Owncast 24/7 online

### DISCUSSION THREAD

**Discussion by SpiderChat** [owncast/owncast#1383] [Q&A]
Dear 

I have runing owncast on my linux server. Everything working fine.

But Owncast  down when i close my ssh.

How to run owncast for 24/7 ?

**Comment by gabek** [comment]
You're probably wanting to run it as a system service using something like systemd under Linux.  A brief overview can be found here:

https://owncast.online/docs/systemservice/

**Comment by controlfreakstudio** [comment]
or if you don't want to run as service you can rock a
./owncast &
